### PR TITLE
Mayhem Heroes Integration

### PR DESCRIPTION
**Pull request by capuanob**

### CODE REVIEW

File: mayhem-fuzz.c [hunk 1]
```diff
@@ -0,0 +1,63 @@
+#include <stdint.h> // uint8_t
```
Comment (xansec) [inline mayhem-fuzz.c:1]: Let's rename this to indicate what's being fuzzed. It's a libfuzzer target, so can be generic (does not have to be mayhem-*).

File: mayhem/Mayhemfile [hunk 2]
```diff
@@ -0,0 +1,7 @@
+image: ghcr.io/capuanob/reliable-fuzz:latest
```
Comment (xansec) [inline mayhem/Mayhemfile:1]: Target name/command names should be descriptive if we have the option. Please omit image name.

**Review by xansec — CHANGES_REQUESTED** [review]
Please make the requested changes.

**Comment by capuanob** [comment]
@xansec Done!

**Pull request merged**